server/seeds/songSeeds.json

```diff
@@ -1,14 +1,50 @@
 [
```

Comment:
Add some seeds scripts to populate User and Ratings as well.

Make sure they follow the proper constraints.